tests/loaddata/test_video_access.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import torch
+from yacs.config import CfgNode as CN
+
+from kale.loaddata.video_access import get_image_modality, VideoDataset, VideoDatasetAccess
+from kale.utils.seed import set_seed
+
+SOURCES = [
+    "EPIC;8;epic_D1_train.pkl;epic_D1_test.pkl",
+    "ADL;7;adl_P_04_train.pkl;adl_P_04_test.pkl",
+    "GTEA;6;gtea_train.pkl;gtea_test.pkl",
+    "KITCHEN;6;kitchen_train.pkl;kitchen_test.pkl",
+]
+TARGETS = [
+    "EPIC;8;epic_D1_train.pkl;epic_D1_test.pkl",
+    "ADL;7;adl_P_04_train.pkl;adl_P_04_test.pkl",
+    "GTEA;6;gtea_train.pkl;gtea_test.pkl",
+    "KITCHEN;6;kitchen_train.pkl;kitchen_test.pkl",
+]
+ALL = SOURCES + TARGETS
+
+IMAGE_MODALITY = ["rgb", "flow", "joint"]
+VAL_RATIO = [0.1]
+seed = 36
+set_seed(seed)
+
+
+@pytest.fixture(scope="module")
+def testing_cfg():
+    cfg = CN()
+    cfg.DATASET = CN()
+    cfg.DATASET.ROOT = "E:/out/python/pt/pykale/tests/test_data/video_data/"
+    cfg.DATASET.IMAGE_MODALITY = "joint"
+    cfg.DATASET.FRAMES_PER_SEGMENT = 16
```

Comment:
Is 16 the minimally possible (for the file size)? Will 8 or 4 work but lead to smaller file size?